tests/services/test_service.py

```diff
@@ -0,0 +1,52 @@
+import os
+
+from sflvault.services.service import Service
+
+test_server_url = os.getenv("VAULT_SERVER")
+json_data = '{"id":1,"created":"2018-08-13","updated":"2018-08-13","url":"https://vault.server/","metadata":{},"notes":"Test","secret":"Test","service_group":1,"parent":2}'
+service = Service()
+service_id = 1
+
+
+def test_get_service_list(requests_mock):
+    requests_mock.get("{}{}".format(test_server_url, service.base_url), json=json_data)
```

Comment:
Let's use `urllib.parse.urljoin` for these (with a graceful fallback to import the proper function in Python 2.x, of course) so that we can anticipate errors in adding too many or too few slashes in the variables.